.github/workflows/e2e.yml

```diff
@@ -31,6 +31,22 @@ jobs:
             echo "is_pr=false" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Add comment to PR
+        if: steps.check_pr.outputs.is_pr == 'true'
+        uses: actions/github-script@60a0d83 # v7.0.1
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const issue_number = ${{ github.event.inputs.pr_or_branch }};
+            const repository = context.repo;
+            const url = `https://github.com/${repository.owner}/${repository.repo}/actions/runs/${process.env.GITHUB_RUN_ID}`;
+            const message = `e2e workflow launched on this PR: [View run](${url})`;
+            github.rest.issues.createComment({
```

Comment:
Why not just use the `gh` command available on the runner? 

`gh pr comment <prnumber> --body "<message>"`

Then you don't need to use actions/github-script. It can all be done in bash.

Reply:
ok.